apps/www/components/Frame/Header.js

```diff
@@ -190,9 +190,24 @@ const Header = ({
                 )}
                 inNativeIOSApp={inNativeIOSApp}
                 onClick={() => {
-                  if (router.asPath === userButtonLink) {
-                    closeHandler()
+                  if (router.asPath.startsWith(userButtonLink)) {
+                    return closeHandler()
                   } else {
+                    if (userButtonLink === '/anmelden') {
+                      try {
+                        const url = new URL(
+                          userButtonLink,
+                          window.location.origin,
```

Comment:
Not 100% sure but this will construct a full URL, but when using `router.push`, a relative one is preferable?

So this could be simplified to
```
router.push(`${userButtonLink}?redirect=${encodeURIComponent(router.asPath)}`)
```